docs/tutorial/application-debugging.md

```diff
@@ -36,3 +36,27 @@ For more information, see the [Debugging the Main Process documentation][main-de
 [node-inspect]: https://nodejs.org/en/docs/inspector/
 [devtools]: https://developer.chrome.com/devtools
 [main-debug]: ./debugging-main-process.md
+
+## V8 Crashes
+
+If the V8 context crashes, the DevTools will display this message.
+
+`DevTools was disconnected from the page. Once page is reloaded, DevTools will automatically reconnect.`
+
+Chromium logs can be enabled via an environment variable.
+
+POSIX shell example:
+
+```sh
+$ export ELECTRON_ENABLE_LOGGING=true
```

Comment:
Let's instead point to [existing documentation](https://github.com/electron/electron/blob/83124889e5dabbc5c67c41d43c5dc9d5b38f921e/docs/api/environment-variables.md#L119) for this env var and move the [existing examples](https://github.com/electron/electron/blob/83124889e5dabbc5c67c41d43c5dc9d5b38f921e/docs/api/environment-variables.md#L8-L20) into it.

Reply:
Sure. Given that the example is the top level example, should the Environment Variables docs be left as is?

Reply:
what's there now is good imo!